### DISCUSSION TITLE
Tests are not discovered on VS for Mac

### DISCUSSION THREAD

**Discussion by liorbalmas** [abpframework/abp#7438] [Q&A]
Hi, 
Test runner isn't finding tests.
My VS on Mac Version is 8.9 Preview (8.9 build 964).

Does anyone here successfully run tests on a similar environment?
Thank you.

**Comment by wocar** [comment]
Hello I'm using mac too. So far I dont have any trouble. Maybe you could post your SDK versions and ABP version.

**Reply by liorbalmas** [reply]
abp 4.1.1
.net 5.0
